module/data/shared/roll-config-field.mjs

```diff
@@ -27,6 +26,13 @@ export default class RollConfigField extends foundry.data.fields.SchemaField {
       }),
       ...fields
     };
+    if ( ability !== false ) {
+      fields.ability = new StringField({
+        required: true,
+        initial: ability,
+        label: "DND5E.AbilityModifier"
+      });
+    }
```

Comment:
You can follow this pattern which we have used in `MovementField` and elsewhere to allow for deleting fields (add the field back above of course). I find it a bit more readable to define all of the fields in the same way:

```suggestion
   Object.entries(fields).forEach(([k, v]) => !v ? delete fields[k] : null);
```